config/rbac/role.yaml

```diff
@@ -8,14 +8,30 @@ rules:
 - apiGroups:
   - core.rukpak.io
   resources:
-  - bundledeployments
+  - deployments
```

Comment:
I'm fine with changing this marker, if we want to specifically watch bundle deployments.